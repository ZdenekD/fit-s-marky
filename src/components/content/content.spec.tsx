import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Content from '.';

describe('Content', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Content>Content</Content>);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-content');

        expect(component.exists()).toBe(true);
    });
});
