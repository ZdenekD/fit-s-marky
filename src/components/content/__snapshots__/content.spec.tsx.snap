// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content match snapshot 1`] = `
<section class="section "
         data-test="component-content"
>
  Content
</section>
`;
